### PR TITLE
Alternate PR for Special Character Remover plugin

### PR DESCRIPTION
**Pull request by mattyb149**
*** NOTE *** Do not merge this as-is! The URL refers to a raw ZIP file on GitHub, and as such will not download properly (they use redirects and such to prevent hosting ZIPs and such).

This is meant as a replacement for #269 as the rebase there was unsuccessful.  

@rishuatgithub, please host your ZIP somewhere else and comment here with its URL, as well as whether everything else about your marketplace entry looks ok.  Once the download works and you've approved the entry, I will merge this.

Thank you for your contribution!

### CODE REVIEW

File: marketplace.xml [hunk 1]
```diff
@@ -7180,4 +7180,41 @@ Pentaho currently does not provide a mechanism allowing for plugin steps to be d
...
+    </installation_notes>
+    <documentation_url></documentation_url>
+    <versions>
+      <version>
+        <branch>stable</branch>
+        <version>1.1.0</version>
+        <build_id/>
+        <name>Version 1.1.0</name>
+    <min_parent_version>4.8</min_parent_version> 
+        <package_url>https://github.com/rishuatgithub/Pentaho/blob/master/SpecialCharacterRemover/marketplace/SpecialCharacterRemover-ver-1.1.0.zip?raw=true
```
Comment (rishuatgithub) [inline marketplace.xml]: https://github.com/rishuatgithub/Special-Character-Remover/releases/download/SpecialCharacterRemover_v1.1.0/SpecialCharacterRemover-ver-1.1.0.zip

**Comment by rishuatgithub** [comment]
@mattyb149 As requested i have placed the file in a new location. The download is working fine. 

URL : https://github.com/rishuatgithub/Special-Character-Remover/releases/download/SpecialCharacterRemover_v1.1.0/SpecialCharacterRemover-ver-1.1.0.zip

Do i need to create a seperate PR again?. The marketplace entry xml needs to be modified in the <package_url/> tag. Rest of the doc. is fine.

**Comment by rishuatgithub** [comment]
@mattyb149 hi.. can you please close this request. I will create a new PR with an updated marketplace xml. You can have a check and merge it, if its fine.

**Comment by mattyb149** [comment]
Closing in favor of #349

**Pull request closed without merging**